p2p/src/peers.rs

```diff
@@ -353,7 +245,7 @@ impl Peers {
 	/// Ping all our connected peers. Always automatically expects a pong back
 	/// or disconnects. This acts as a liveness test.
 	pub fn check_all(&self, total_difficulty: Difficulty, height: u64) {
-		for p in self.connected_peers().iter() {
+		for p in self.peers_iter().connected() {
```

Comment:
Given that these functions are defined on the `Peer` struct, would it make sense to remove the "peers" word from them? So `iter()` instead of `peers_iter()`